run-style-checks.sh

```diff
@@ -7,6 +7,6 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 pydocstyle invenio_app_ils tests docs && \
-isort -rc -c -df && \
+isort --check-only --diff && \
```

Comment:
Missed the recursive part
```suggestion
isort --recursive --check-only --diff && \
```

Reply:
In isort v5 `--recursive` option has been removed. Directories passed in are now automatically sorted recursive.